index.js

```diff
@@ -103,6 +132,11 @@ app.get("/v1/:username", async (req, res) => {
   }
 });
 
+app.post("/twitter", (req, res) => {
```

Comment:
can we change the path to `/v1/uploadToTwitter`?

Reply:
@sallar what do you think about `/v1/shareMediaTweet` or `/v1/tweetMedia`

Reply:
Yeah both are nice

Reply:
@sallar @catalinmiron 

ok. change to `/v1/tweetMedia` :)